src/esp/nav/PathFinder.cpp

```diff
@@ -519,6 +519,10 @@ bool PathFinder::build(const NavMeshSettings& bs,
       return false;
     }
 
+    if (navMesh_) {
+      dtFree(navMesh_);
```

Comment:
I think this should be dtFreeNavMesh. Otherwise, the destructor won't get called.